metabase/metabase
# feat(sdk): ability to pass parameters to static questions

Closes https://github.com/metabase/metabase/issues/45343

### Description

Adds the `parameterValues` property to static questions, to allow passing parameter values by their parameter slug:

```tsx
<StaticQuestion
  questionId={161}
  showVisualizationSelector={false}
  height={questionHeight}
  parameterValues={{ product_id: product.id }}
/>
```

### How to verify

- Go to the `sdk-parameterized-static-questions` branch in Shoppy
- Add the `parameterValues` field to static questions
- SQL questions should now take on parameter values

### Demo

Using `parameterValues={{ product_id: product.id }}` on [question 161 on Shoppy](https://shoppy.hosted.staging.metabase.com/question/161-order-quantity-by-product-month)

![CleanShot 2567-07-13 at 00 05 05@2x](https://github.com/user-attachments/assets/85b7b5af-0858-49b6-b475-42511e0d3bbe)

### Checklist
